basics/basics.md

```diff
@@ -18,9 +18,14 @@ You might say that the OHM floor price or intrinsic value is 1 DAI. We believe t
 
 ## How does it work?
 
-At a high level, OlympusDAO consists of its protocol managed treasury, protocol owned liquidity, bond mechanism, and high staking rewards that are designed to control supply expansion.
+At a high level, OlympusDAO consists of its protocol managed treasury, protocol
+owned liquidity \([POL](../references/glossary.md#POL)\), bond mechanism, and high
```

Comment:
Maybe as an idea let's not emphasize "high" this much anymore. We will gradually adjust the reward rate from here. 

```suggestion
owned liquidity \([POL](../references/glossary.md#POL)\), bond mechanism, and
```